snuba_sdk/legacy.py

```diff
@@ -134,10 +135,17 @@ def to_scalar(value: Any) -> Any:
         limit, name = limitby
         query = query.set_limitby(LimitBy(Column(name), int(limit)))
 
-    extras = ("limit", "offset", "granularity", "totals")
+    extras = (
+        "limit",
+        "offset",
+        "granularity",
+        "totals",
```

Comment:
How does TOTALS work in a joined query ?

Reply:
From reading it seems like it work the same as a normal query.